ownCloud/Viewer/DisplayViewController.swift

```diff
@@ -307,11 +310,24 @@ class DisplayViewController: UIViewController {
 		self.noNetworkLabel?.isHidden = true
 		self.showPreviewButton?.isHidden = false
 	}
+
+	@objc func optionsBarButtonPressed() {
+
+		let actionsLocation = OCExtensionLocation(ofType: OCExtensionType.action, identifier: nil)
```

Comment:
Please add the location identifier here, too:
```suggestion
		let actionsLocation = OCExtensionLocation(ofType: .action, identifier: .moreItem)
```